### ISSUE TITLE
Doxbin is detecting SA 1.5.9

### ISSUE THREAD

**Issue by ctaity** [ulixee/secret-agent#348]
HI, doxbin.org is still detecting SA.

![Screen Shot 2021-09-08 at 19 05 58](https://user-images.githubusercontent.com/5207990/132592029-cb1135b5-ec08-4d28-8376-5d63f5db4dde.png)

**Comment by blakebyrnes** [comment]
I didn’t get that message. Are you sure it’s not actually because of an extension you have installed? Doxbin just spun for me, but no error.

**Comment by ctaity** [comment]
that screen is from SA BROWSER: SA_SHOW_BROWSER =true

**Comment by blakebyrnes** [comment]
Can you send me a session db? Maybe I can find something in it

**Comment by blakebyrnes** [comment]
What does this site do in regular chrome? I can’t get it to do anything in any browser I’ve tried besides bounce the logo

**Comment by ctaity** [comment]
on a regular browser, pass the checks and go to the site: 

![Screen Shot 2021-09-08 at 16 13 48](https://user-images.githubusercontent.com/5207990/132600273-4198334e-a1e6-43d0-becc-3411834b4b3d.png)

**Comment by ctaity** [comment]
and CloudFlare is still detecting SA:

https://www.loom.com/share/b4e64a724d1e4ca098d46a824c4842a4

**Comment by blakebyrnes** [comment]
I need more to go on than just urls. Please either provide session databases (private is fine) or way more details. I’m not sure I follow how a 522 error on voat.co is related to doxbin.

**Comment by ctaity** [comment]
no voat.co was an error, please excuse me. Tomorrow i will send the session database. Thanks for all

**Comment by ctaity** [comment]
@blakebyrnes  session db 
[Uploading Archive.zip…]()

**Comment by blakebyrnes** [comment]
Yep, thanks! I poked through, but I don't see anything obvious. It looks like their bot blocker is seeing something it doesn't like, but it's not showing up in the logs in any way I can see. Do you know if any other tools are making it through on this site?

**Comment by ctaity** [comment]
No, I am tried with selenium, puppeteer with stealth plugin, and normal requests but not lucky, and with SecreatAgent :(

**Comment by SleepTheGod** [comment]
![image](https://github.com/user-attachments/assets/a6934440-4eff-4765-8fe5-610ed0d2aed3)
looks up to me for now skids

**Comment by SleepTheGod** [comment]
It's because they have no idea how to build a proper flask application

**Comment by SleepTheGod** [comment]
![1841780094584438847-GY9RNg1WQAERwkG](https://github.com/user-attachments/assets/0665e8ab-d236-4026-a374-864cf20a5d03)

**Comment by SleepTheGod** [comment]
Tell kt i said i hope they like prison

**Comment by SleepTheGod** [comment]
LMAO YOU PEOPLE CANT EVEN DEV A SITE PROPERLY HAHAHAHAHAHAHAHAHA
